### PR TITLE
fixes #8817

### LINKED ISSUE #8817: Borers can drag people
this is old but apparently it wasn't reported before

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #8817

### CODE REVIEW

File: code/modules/mob/living/simple_animal/borer.dm [hunk 1]
```diff
@@ -223,6 +223,10 @@ var/global/borer_unlock_types = typesof(/datum/unlockable/borer) - /datum/unlock
 
 		stat("Chemicals", chemicals)
 
+/mob/living/simple_animal/borer/start_pulling(var/atom/movable/AM)
+	to_chat(src, "<span class='warning'>You are too small to pull anything.</span>")
+	return
```
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer.dm]: WHY

Comment (Intigracy) [inline code/modules/mob/living/simple_animal/borer.dm]: WHY what?

Comment (Intigracy) [inline code/modules/mob/living/simple_animal/borer.dm]: I mean, this is exactly how mouse does it.

Would you rather I make another mob var and have it checked in the parent for start_pulling?

Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer.dm]: I'm talking about the trailing return.

Comment (Intigracy) [inline code/modules/mob/living/simple_animal/borer.dm]: o

**Comment by Intigracy** [comment]
There ya go fam.

**Comment by unid15** [comment]
maybe make it size bazed

if a mob's size is 1 it cant pull anything

**Pull request merged**